packages/mask/src/extension/background-script/EthereumServices/rpc/index.ts

```diff
@@ -132,7 +133,7 @@ export async function getSendTransactionComputedPayload(payload: JsonRpcPayload)
         }
 
         // contract deployment
-        if (isSameAddress(to, ZERO_ADDRESS)) {
+        if (isZeroAddress(to)) {
```

Comment:
No need to remove the importing of `ZERO_ADDRESS`.

Reply:
https://github.com/DimensionDev/Maskbook/blob/7c00da4c23bfb576a090b8a6b35137cbc2c54208/packages/mask/src/extension/background-script/EthereumServices/rpc/index.ts#L32-L37